function-web-model/src/main/java/com/future/function/web/model/request/communication/ChatroomRequest.java

```diff
@@ -18,6 +18,8 @@
   @ChatroomName(message = "InvalidName")
   private String name;
 
+  private String picture;

```

Comment:
Could add `@FileMustExist` annotation for validation though, but the data type must be changed to list and add `@Size(1)` as well to prevent more than one resource id being passed

Reply:
Ok, will be updated in next commit